src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -565,6 +567,8 @@ private async Task SyncPackageDisplayServiceName(IEnumerable<ReviewModel> review
                     review.PackageDisplayName = newDisplayName;
                     await _reviewsRepository.UpsertReviewAsync(review);
                 }
+                // Wait before processing next review to delay background task
+                await Task.Delay(1000);
```

Comment:
Do we need the delay still given you moved it to the end of the background processing?

Reply:
I think it's still better to add a delay to avoid continuous run of this background thread.